README.md

```diff
@@ -26,7 +26,9 @@
 👤 **IJeoma Odinko**
 - GitHub: [@ijeomaese](https://github.com/ijeomaese)
 - Twitter: [@iodinko](https://twitter.com/Iodinko)
-- LinkedIn: [Ijeoma Odinko(https://www.linkedin.com/in/ijeoma-odinko
+- LinkedIn: [Ijeoma Odinko(https://www.linkedin.com/in/ijeoma-odinko)
+- Website:  https://ijeomaodinko.github.io/Portfolio-Page/
```

Comment:
- [ ] The link you are given is not working. Please kindly re-deploy the project to Github and add the working version link 👍
- [ ] Let's also move the live demo link to the top of the readme because it shouldn't be in your about section.

Example 
![image](https://user-images.githubusercontent.com/69011963/160432132-4dd2eb5a-33bd-4ad2-9483-efc87d154d15.png)